code/game/turfs/simulated/floor/asteroid_floors.dm

```diff
@@ -166,6 +166,15 @@
 	color = COLOR_FLOOR_HARD_ROCK
 	baseturf = /turf/simulated/floor/lava/lava_land_surface
 
+/turf/simulated/floor/plating/asteroid/basalt/plasma
+	temperature = 1000
+	oxygen = 0
+	nitrogen = 0
+	toxins = 10
+	agent_b = 0.1
```

Comment:
I smell..... gaming.

Reply:
Im not sure I understand, is the issue with the Agent B? It was included in my proposal and hasnt had any comments about the inclusion as of yet

Reply:
Yeah I'm not sure how I feel about having sources of Agent B on the station in a readily accessible form currently, it's rare for a reason as it can easily cause major self-sustaining fires. I totally missed it in the proposal and I assume others did too in their focus on other elements, the fact it's not mentioned on the PR doesn't help.
Might have to be pepared to review the inclusion of this.

Reply:
after discussion, its been removed. if its to be added back ill make a separate PR and proposal